### DISCUSSION TITLE
Bootstrap?

### DISCUSSION THREAD

**Discussion by Laanan** [opencardev/crankshaft#545] [Q&A]
Hi there,

I'm wondering if crankshaft has been tried with non-rpi linux head units. In other words, can I use crankshaft as a boostrap solution within an existing linux filesystem? I've been tinkering with what seems to be linux based firmware for mine with the end goal of providing screen mirroring (my head unit was EOL as soon as it left the factory floor, due to the emergence of Android Auto and Carplay). Any advice would be great. Thanks!

**Comment by jroden2** [comment]
This is something I'm quite interested in, I have a car running Window CE from the factory. Thinking I could port this, maybe an AGL install with crankshaft bootstrapped...

**Comment by matt2005** [comment]
no, i've not tried it. I'd be willing to try it but don't have any hardware to try.
